docs/plugins/asg_rollout.md

```diff
@@ -0,0 +1,92 @@
+# Auto-Scaling Group LaunchConfig Rollout Tool
+
+## Overview
+
+This plugin adds support for rolling out changes to Auto Scaling
+Groups that have happened after a stack update. It supports various
+pre- and post-actions on the instances (using Moonshot's native SSH
+support).
+
+## Example
+
+The ASGRollout class is intended to be used within a method in your
+project's derived class of the Moonshot::CLI class.
+
+```ruby
+#!/usr/bin/env ruby
+
+require 'moonshot'
+
+class MyService < Moonshot::CLI
+  # .. normal configuration ..
+
+  desc :asg_rollout, 'Update instances in the Auto Scaling Group'
+  def asg_rollout
+    Moonshot::Tools::ASGRollout.new(
```

Comment:
Swapping this for a Config object to line up with other Moonshot internals, and to be sane.